survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/question_parts/DoubleTextFieldPart.kt

```diff
@@ -0,0 +1,24 @@
+package com.quickbirdstudios.surveykit.backend.views.question_parts
+
+import android.content.Context
+import android.text.InputType
+import android.view.View
+import androidx.annotation.StringRes
+import com.quickbirdstudios.surveykit.R
+
+internal class DoubleTextFieldPart(context: Context) : TextFieldPart(context) {
+
+    init {
+        id = R.id.doubleFieldPartField
+        this.field.inputType = InputType.TYPE_CLASS_NUMBER or InputType.TYPE_NUMBER_FLAG_DECIMAL
+        this.textAlignment = View.TEXT_ALIGNMENT_CENTER
+    }
+
+    companion object {
+        fun withHint(context: Context, @StringRes hint: Int) = DoubleTextFieldPart(
+            context
+        ).apply {
+            field.hint = context.getString(hint)
+        }
+    }
+}
```

Comment:
I think after applying `ktlint` there should be a new line here. Are you sure you applied it in all files?

Reply:
![Capture d’écran, le 2020-08-11 à 17 32 04](https://user-images.githubusercontent.com/4621982/89950977-9bc56a00-dbf8-11ea-90b7-034ae929d006.png)

Ok this is weird, the empty lines at the end of files are still here in my code (I opened the file with TextEdit and it's still here) but it looks like Github got rid of them. I ran Ktlint on all the project.

Reply:
Oh sorry I see what you meant by applying it, not juste the lint, the format